src/Parlot/Fluent/ParseResult.cs

```diff
@@ -31,6 +31,6 @@ public void Set(string buffer, in TextPosition start, in TextPosition end, strin
     
     public static class ParseResultExtensions
     {
-        public static ReadOnlySpan<char> GetSpan<T>(this ParseResult<T> result) => result.Buffer.AsSpan(result.Start.Offset, result.End - result.Start);
+        public static ReadOnlySpan<char> GetSpan<T>(this in ParseResult<T> result) => result.Buffer.AsSpan(result.Start.Offset, result.End - result.Start);
```

Comment:
IIRC `in` shouldn't be used with regular structs, only with read-only ones. `ParseResult` is regular.

Reply:
@lahma good catch! I'll remove this one!

Reply:
This whole method/class can be removed. And the `Buffer` in `ParseResult` too.

Reply:
But it's out of the scope of this PR

Reply:
Yeah I saw it wasn't used in Parlot but didn't check if you were using it outside of it.